lesson_15/lesson_15_Turcu_Alexandru/unittest.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from lesson_15.lesson_15_Turcu_Alexandru import homework
+
+
+class TestSquareArea(unittest.TestCase):
+    def test_area(self):
+        elem = 2
+        area = elem * elem
+        self.assertAlmostEqual(area, elem*elem)
+
+    def test_square_type(self):
+        test_homework1 = homework.Polygons(1, 2, 3)
+        self.assertIsInstance(1, int)
```

Comment:
same as above, this test will never fail, 1 will always be an int. use the code an test on test_homework1.sides[0]

Reply:
So basicly like this , I understand now !  self.assertIsInstance(test_homework1.sides[0], int)

Reply:
Yes, that is correct, can you also make the changes? The following tests need some changes 

- `test_area` you are not using the code  in test
- `test_square_type` you are not using the code in test
- `test_polygon_perimeter` you are not using the code perimeter should be equal to test_homework2.perimeter()
- `test_sides` not using code, you make the assert on sides with you defined in the test, you need to make the assert on the result that the code gives you